logging/bin/deploy_logging_open.sh

```diff
@@ -86,7 +86,13 @@ set -e
 # Version Info                   #
 ##################################
 
-if ! deployV4MInfo "$LOG_NS"; then
+# If a deployment with the old name exists, remove it first
+if helm3ReleaseExists v4m $LOG_NS; then
+  log_verbose "Removing outdated instance of SAS Viya Monitoring"
+  helm uninstall -n "$LOG_NS" "v4m"
+fi
+
+if ! deployV4MInfo "$LOG_NS" "v4m-log"; then
```

Comment:
Let's go with "v4m-logs" rather than "v4m-log"...make it plural as you did with the metrics one...(and we can do "traces" if/when we add tracing).

Reply:
Updated the names to v4m-logs.  It should show up in the next push.